README.md

```diff
@@ -193,7 +193,7 @@ helm show values hedera-json-rpc-relay
 Deploy an installation with custom values file:
 
 ```
-helm install custom-hedera-json-rpc-relay -f path/to/values/file.yaml ./helm-chart --debug
+helm install custom-hedera-json-rpc-relay -f path/to/values/file.yaml ./charts --debug
```

Comment:
This example seems to be an outdated reference from before the refactoring, should probably be `./charts/hedera-json-rpc`

Reply:
Thanks, will fix in follow up